required/latex-lab/testfiles-math-luatex/mathml-luamml-5.lvt

```diff
@@ -0,0 +1,25 @@
+\ExplSyntaxOn
+\sys_gset_rand_seed:n{1000}
+\ExplSyntaxOff
+\DocumentMetadata{testphase={phase-III,math}, pdfversion=2.0, 
```

Comment:
not really important but we should start using testphase=latest in my opinion, that is helping in the furture if new functionaly is breaking stuff, that might otherwise be masked by loading only a portion of the code.

Reply:
done